src/main/groovy/org/jenkinsci/plugins/globalEventsPlugin/GlobalItemListener.java

```diff
@@ -0,0 +1,80 @@
+package org.jenkinsci.plugins.globalEventsPlugin;
+
+import hudson.Extension;
+import hudson.model.Item;
+import hudson.model.listeners.ItemListener;
+import jenkins.model.Jenkins;
+
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+/**
+ * Created by Renjith Pillai (i306570) on 15/09/16.
```

Comment:
And I'm not sure that we are keeping information about creator/ownership in source code itself. Git shows it much better.